in/install.sh.in

```diff
@@ -15,17 +14,11 @@ main() {
     fi
 
     mkdir -p "${LIB_DIR}"
-    cp "lib/${NAME}.jar" "${LIB_DIR}/${NAME}.jar"
-
-    mkdir -p "${BIN_DIR}"
-    cat <<EOF > "${BIN_DIR}/start"
-#!/usr/bin/env bash
-java -cp "${LIB_DIR}/${NAME}.jar" clojure.main -m ptc.start
-EOF
-    chmod +x "${BIN_DIR}/start"
+    cp "lib/${NAME}.jar" "${LIB_DIR}/${NAME}"
```

Comment:
Can you keep the file extension please since this isn't really a binary?

Reply:
replaced .jar